test/python/circuit/test_initializer.py

```diff
@@ -39,6 +39,36 @@ class TestInitialize(QiskitTestCase):
 
     _desired_fidelity = 0.99
 
+    def test_disentangled(self):
```

Comment:
Please note that the current Qiskit [`Initialize`](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.Initialize) code is based on [`Isometry`](https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.library.Isometry) by Iten et al. [Phys. Rev. A 93, 032318](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.93.032318)

Reply:
Thanks, we fixed the optimization and now the test works.